src/taglibs/migrate/all-tags/dynamic-attributes.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function migrate(el, context) {
+    el.forEachAttribute(attr => {
+        const name = attr.name;
+        const value = attr.value;
+        if (
+            name == undefined &&
+            (value.type == "MemberExpression" || value.type == "Identifier")
+        ) {
+            context.deprecate(
+                'The "${attributes}" is deprecated. Please use "...attributes" modifier instead. See: https://github.com/marko-js/marko/wiki/Deprecation:-w-*-Atrributes'
+            );
+            attr.spread = true;
```

Comment:
This line can be removed if we switch to removing the attribute and adding the spread attribute to the end.